package.json

```diff
@@ -41,8 +41,11 @@
 	"dependencies": {
 		"bits-ui": "^0.21.12",
 		"clsx": "^2.1.1",
+		"lucide-svelte": "^0.414.0",
```

Comment:
I think we should avoid pulling in this lib. At least not when we have the alternative of using the ones we need from vercel.

Reply:
Yeah no problem we aren't even using it its just there because the normal dialog component uses it.